cmd/download_test.go

```diff
@@ -25,9 +25,7 @@ func TestDownloadWithoutToken(t *testing.T) {
 
 	err := runDownload(cfg, pflag.NewFlagSet("fake", pflag.PanicOnError), []string{})
 	if assert.Error(t, err) {
-		assert.Regexp(t, "Welcome to Exercism", err.Error())
-		// It uses the default base API url to infer the host
-		assert.Regexp(t, "exercism.io/my/settings", err.Error())
+		assert.Regexp(t, "there is no token configured", err.Error())
```

Comment:
Why does this change?

Reply:
As far as I can see, all tests on `main` are green: https://github.com/exercism/cli/actions/runs/6098825660/job/16549554194

Reply:
According to [CONTRIBUTING.MD](https://github.com/exercism/cli/blob/main/CONTRIBUTING.md) you can run tests locally with `go test ./...` and this fails on a clean clone and `master`. I get this:

```
pwd && go test ./...
/Users/baduker/Downloads/cli
?   	github.com/exercism/cli/browser	[no test files]
?   	github.com/exercism/cli/exercism	[no test files]
ok  	github.com/exercism/cli/api	0.254s
ok  	github.com/exercism/cli/cli	0.315s
--- FAIL: TestSubmitFiles (0.01s)
        Error Trace:    submit_test.go:275
        Error:		Not equal: "This is file 2." (expected)
        	        != "" (actual)
        
--- FAIL: TestSubmitFilesForTeamExercise (0.00s)
        Error Trace:    submit_test.go:471
        Error:		Not equal: "This is file 2." (expected)
        	        != "" (actual)
        
FAIL
FAIL	github.com/exercism/cli/cmd	0.529s
ok  	github.com/exercism/cli/config	0.370s
ok  	github.com/exercism/cli/debug	0.424s
ok  	github.com/exercism/cli/workspace	(cached)
FAIL
```

Reply:
Do you mean `master` or `main`? The primary branch on this repo is `main`.

Reply:
Yes, I meant `main`, not `master`.

Reply:
Hmmm, I don't know why that's not working on your machine when it is working on our CI. Maybe a different version of Go? CCing @junedev, @andrerfcsantos, @kytrinyx and @ErikSchierboom to see if they have any ideas :)

Reply:
@iHiD @ErikSchierboom the failing tests happen when using a version of Go 1.17 and above. There was a change to the [mime/multipart pkg ](https://go.dev/doc/go1.17#mime/multipart) that changes the file path information being returned any files included in the request. It looks like there's an open PR for this fix https://github.com/exercism/cli/pull/1066 that can be merged in.

Reply:
I've replied there. I'd appreciate eyes on that too pls :)

https://github.com/exercism/cli/pull/1066/files#r1331620696